docs/wiki-links.md

```diff
@@ -16,72 +16,15 @@ Foam enables you to Link pages together using `[[file-name]]` annotations.
 
 ## Markdown compatibility
 
-The [Foam for VSCode](https://marketplace.visualstudio.com/items?itemName=foam.foam-vscode) extension automatically generates [markdown link reference definitions](https://spec.commonmark.org/0.29/#link-reference-definitions) at the bottom of the file to make wiki-links compatible with Markdown tools and parsers.
-
-If you look at link references the bottom of any Foam workspace file that uses wiki-links, you should see an automatically generated list of references that look as follows:
-
-```markdown
-[wiki-links]: wiki-links "Wiki Links"
-[other-page]: other-page "Other Page"
-```
-
-These exist to make `[[wiki-links]]` compatible with Markdown-consuming tools such as static site generators, VS Code plugins etc. 
-
-## Why don't `[[wiki-links]]` work on GitHub
```

Comment:
Note to self: When this change lands to docs, go and update a bunch of issues where this document is referenced.